app/templates/components/public/side-menu.hbs

```diff
@@ -17,25 +17,22 @@
     <a class="item" href="{{href-to 'public.index'}}">
       {{t 'Info'}}
     </a>
-    {{#if event.isTicketingEnabled}}
+    {{#if event.tickets.length}}
       <a class="item" href="{{href-to 'public.index'}}#tickets">
         {{t 'Tickets'}}
       </a>
     {{/if}}
-    {{#if event.isSessionsSpeakersEnabled}}
+    {{#if event.speakers.length}}
       <a class="item" href="{{href-to 'public.index'}}#speakers">
         {{t 'Speakers'}}
       </a>
     {{/if}}
   {{/if}}
-  {{#if event.isSessionsSpeakersEnabled}}
+  {{#if event.sessions.length}}
     {{#link-to 'public.sessions' class='item'}}
       {{t 'Sessions'}}
     {{/link-to}}
   {{/if}}
-  {{#link-to 'public.schedule' class='item'}}
-    {{t 'Schedule'}}
-  {{/link-to}}
```

Comment:
Why delete this ?

Reply:
We do not have support to add schedule for a event. We just get a empty page when clicked on this. In our meeting we were told to remove it temporarily.

Reply:
alright.